LondonBritishLibrary/orient/BLorient546.xml

```diff
@@ -31,21 +31,165 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.7">
+<locus from="101v"/>
+<title type="complete" xml:lang="gez" ref="LIT1347Epipha">Of Epiphanius</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.8">
+<locus from="107v"/>
+<title type="complete" xml:lang="gez" ref="LIT1103Anapho">Of the 318 Nicene fathers</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.9">
+<locus from="115v"/>
+<title type="complete" xml:lang="gez" ref="LIT2040Naakkw">Of Gregory of Alexandria <sic/></title>
+<incipit xml:lang="gez"></incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.10">
+<locus from="121r"/>
+<title type="complete" xml:lang="gez" ref="LIT3180Anapho">Of Dioscorus</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.11">
+<locus from="123r"/>
+<title type="complete" xml:lang="gez" ref="LIT2404Tanseu">Of Jacob of Sǝrug</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.12">
+<locus from="128v"/>
+<title type="complete" xml:lang="gez" ref="LIT1098Anapho">Of Basil of Caesarea</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.13">
+<locus from="137r"/>
+<title type="complete" xml:lang="gez" ref="LIT2038Naakkw">Of Gregory of Armenia</title>
+<incipit xml:lang="gez">ነአኵተከ፡ እግዚኦ፡ ንሴብሐከ፡ ወንዌድሰከ፡ ቡሩክ፡ ስምከ፡ ወንባርከከ፡ ንሕነኒ፡ ውዱስ፡ ስምከ፡</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3.14">
+<locus from="140r"/>
+<title type="complete" xml:lang="gez" ref="LIT1569habeka">Of Cyril of Alexandria</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i4">
+<locus from="146r"/>
+<title type="complete" ref="LIT4942Sarawita"/>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i5">
+<locus from="147v"/>
+<title type="complete" xml:lang="en">Prayers</title>
+<textLang mainLang="gez"></textLang>
+
+<msItem xml:id="ms_i5.1">
+<locus from="147v"/>
+<title type="complete" ref="LIT2252salota">Prayer of the incense</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i5.2">
+<locus from="152v"/>
+<title type="incomplete" xml:lang="en" ref="LIT2167Parali">Prayer of Jeremiah</title>
+<textLang mainLang="gez"></textLang>
+<note>It is a fragment of Paralipomena of Jeremiah.</note>
+</msItem>
+
+<msItem xml:id="ms_i5.3">
+<locus from="152v"/>
+<title type="complete" xml:lang="en">Prayer of supplication</title>
+<incipit xml:lang="gez">ጸሎት፡ ዘተመጥዎ፡ ስእለት፡ እግዚአብሔር፡ አቡሁ፡ ለእግዚእነ፡ ወመድኃኒነ፡ ኢየሱስ፡ ክርስቶስ፡ ወመንፈስ፡ ቅዱስ። ስማዕ፡ ስእለትየ፡
+ወፈጸም፡ ሊተ፡ ኵሎ፡ ፈቃድየ፡ </incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i5.4">
+<locus from="153r"/>
+<title type="complete" xml:lang="en">Prayer of the incense</title>
+<textLang mainLang="gez">ጸሎተ፡ ዕጣን። በእንተ፡ እለ፡ ኖሙ። ኦእግዚኦ፡ ግበር፡ ተካረከ፡ ሠናየ፡ ለአግብርቲከ፡ መሃይምናን፡</textLang>
+</msItem>
```

Comment:
I have found this prayer in: https://betamasaheft.eu/manuscripts/BAVet66/main
msItem 2, however, not in the ascribed textual unit: https://www.digitale-sammlungen.de/de/view/bsb10199036?page=378,379
Maybe, it would worth to open an issue for creation of a new id...